clap_complete/tests/testsuite/powershell.rs

```diff
@@ -132,3 +132,31 @@ fn subcommand_last() {
         name,
     );
 }
+
+#[test]
```

Comment:
If the tests can't be added in a prior commit, then they should be added in the commit with the feature.